README.md

```diff
@@ -1,42 +1,74 @@
-
 Integration Tests for Linux System Roles
 ========================================
 
-A docker container which runs the tests from one of the [linux system
-roles](https://github.com/linux-system-roles) against various operating
-systems.
-
-It starts a virtual machine based on a cloud image of each operating system
-(defined in [images.yml](/images.yml)) and runs playbooks in the roles `test`
-directory against that machine.
+A docker container which runs integration tests for open pull requests on
+[linux system roles](https://linux-system-roles.github.io) repositories. It runs
+all playbooks `test/test_*.yml` of the repository against various virtual
+machines.
```

Comment:
The intent of the standard test invocation spec is to execute ```tests.yml```. Just checking in case that wasn't clear.

Reply:
Yeah, I know. I wanted to have something that works with the repositories that we have now.

But now that you're saying it, it probably makes sense to support both right away. I've changed it to look for `tests/tests.yml` and fall back to the old way only if that doesn't exist.